src/main/java/seedu/loyaltylift/logic/commands/AddCustomerCommand.java

```diff
@@ -18,22 +19,24 @@ public class AddCustomerCommand extends Command {
 
     public static final String COMMAND_WORD = "addc";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. \n"
             + "Parameters: "
+            + "[" + PREFIX_CUSTOMER_TYPE + "ind|ent] "
```

Comment:
should ind|ent be IND|ENT instead? since everything else is capitalised like NAME, PHONE etc